apps/frontend/pages/lems/reports/team-list.tsx

```diff
@@ -45,6 +47,11 @@ const Page: NextPage<Props> = ({ user, division, teams: initialTeams }) => {
       {
         label: `הגעה (${teams.filter(t => t.registered).length}/${teams.length})`,
         sort: 'registration'
+      },
+      {
+        label: 'דף מידע',
+        sort: 'profileDocumentUrl',
```

Comment:
This will cause errors since we have custom sort functions defined and this key isn't one of them